src/main/java/raisetech/studentManagement/date/ApplicationStatus.java

```diff
@@ -0,0 +1,25 @@
+package raisetech.studentManagement.date;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class ApplicationStatus {
+
+  private Integer id;
+
+  @Pattern(regexp = "^\\d+$",message = "数字のみ入力するようにしてください。")
```

Comment:
フォーマットしたら以下の様にスペース入らないですか？ 👀 
```suggestion
  @Pattern(regexp = "^\\d+$", message = "数字のみ入力するようにしてください。")
```

Reply:
全ファイルちゃんとコードフォーマットを実施しましょう